src/parse/rules/go_rules.build_defs

```diff
@@ -705,17 +714,34 @@ def _go_library_cmds(complete=True, all_srcs=False, cover=True, filter_srcs=True
     return cmds
 
 
-def _go_binary_cmds(static=False, ldflags='', pkg_config=''):
+def _go_binary_cmds(static=False, ldflags='', pkg_config='', definitions=None):
     """Returns the commands to run for linking a Go binary."""
     _link_cmd = '$TOOLS_GO tool link -tmpdir $TMP_DIR -extld $TOOLS_LD %s -L . -o ${OUT} ' % _GOPATH.replace('-I ', '-L ')
     prefix = _LINK_PKGS_CMD + _go_import_path_cmd(CONFIG.GO_IMPORT_PATH)
 
+    linkerdefs = []
+    if definitions is None:
```

Comment:
Looks a bit strange to see the pass used here, but I guess it saves a level of indent on using an outer 

```
if definitions:
    if isinstance(definitions, str):
        ...
```

Reply:
This, however, is a python idiom that I definitely picked up: use `pass` responsibly to aid in the readability of code.  This hit one of those use-cases where I was trying to convey to the reader, "yes, I'm deliberately falling through this switch-like statement."